src/main/java/gov/nasa/jpl/mbee/mdk/settings/ProjectSettings.java

```diff
@@ -0,0 +1,141 @@
+package gov.nasa.jpl.mbee.mdk.settings;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.cache.Cache;
+import com.google.common.cache.CacheBuilder;
+import com.nomagic.magicdraw.core.Application;
+import com.nomagic.magicdraw.core.Project;
+import com.nomagic.magicdraw.core.ProjectUtilities;
+import com.nomagic.uml2.ext.jmi.helpers.StereotypesHelper;
+import com.nomagic.uml2.ext.magicdraw.classes.mdkernel.Comment;
+import com.nomagic.uml2.ext.magicdraw.classes.mdkernel.Element;
+import com.nomagic.uml2.ext.magicdraw.classes.mdkernel.Package;
+import gov.nasa.jpl.mbee.mdk.json.JacksonUtils;
+import gov.nasa.jpl.mbee.mdk.util.Utils;
+import org.apache.commons.lang3.StringUtils;
+
+import javax.swing.*;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.TimeUnit;
+
+public class ProjectSettings {
+
+    private static Boolean warnOnce = false;
+
+    private static final Cache<Project, String> PROFILE_SERVER_CACHE = CacheBuilder.newBuilder().weakKeys().maximumSize(100).expireAfterAccess(10, TimeUnit.MINUTES).build();
+
+    public static final String
+            MMS_HOST_URL = "mms.hostUrl",
+            MMS_BASE_PATH = "mms.basePath",
+            VE_HOST_URL = "ve.hostUrl",
+            VE_BASE_PATH = "ve.basePath";
+
+
+    public static ObjectNode getProjectSettings(Project project) {
+        Package model = project.getPrimaryModel();
+        Comment comment = new ArrayList<>(model.getOwnedComment()).get(0);
```

Comment:
Isn't this just equivalent to `Comment comment = model.getOwnedComment()`?

Reply:
No, there could be more than 1 we only want the first

Reply:
Ahh I was misreading it. There shouldn't be a need to instantiate a new `ArrayList` though and this also assumes that there is at least one `ownedComment` without defensively checking. I would recommend using the method you introduced `Utils.getDocumentationElement` (after changing the return type to `Comment`) and handling the null case in this method.

Reply:
OBE